<launch>

    <arg name="config_filepath" default="/home/mike/catkin_ws/src/StairwayDetection/config.yaml"/>
    <arg name="input_cloud_topic" default="cloud"/>
    
    <node pkg="stair_detection" type="stair_detection_node" name="stair_detector" output="screen"
    args=" $(arg config_filepath)">
        <remap from="input_cloud" to="$(arg input_cloud_topic)"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz"
    args="-d /home/mike/.rviz/stair_detection.rviz"/>

</launch>